DEMO.md

```diff
@@ -36,6 +37,8 @@
   - This step will also bring up the individual nodes for tool planning, trajectory planning, 
     perception and robot execution, each in its own terminal window (don't close the windows).
 
+- Run the CNN script in a new window as follows:
+  THEANO_FLAGS='floatX=float32' rosrun gilbreth_perception recognition_cnn.py
```

Comment:
The `application_run.launch` file step above runs the `recognition_cnn.py` node so it's not necessary to run it a second time.  Therefore, the right procedure should probably start with the `THEANO_FLAGS` environment variable declaration followed by the launch file.

Reply:
When I start up using only the application_run.launch file I get the following error: "[recognition_cnn-6] process has finished cleanly" 

When I run the recognition_cnn.py node seperately it does not shut down. I'm not sure why this is but it consistently works.